### ISSUE TITLE
IOT instruction (core dumped)

### ISSUE THREAD

**Issue by samuelortion** [angeluriot/Fractal_generator#2]
Hello,

I really enjoyed your video.

I tried to run the app on my Linux computer, but, after proper build when I launch the application: it shows up and disappear quickly with these errors:

```text
Failed to compile fragment shader:
0:39(74): error: no function with name 'texture2D'
0:39(61): error: operands to arithmetic operators must be numeric
0:39(23): error: operands to arithmetic operators must be numeric
0:63(19): warning: `initial_color' used uninitialized
0:68(19): warning: `initial_color' used uninitialized
0:77(68): warning: `initial_color' used uninitialized
0:81(15): warning: `initial_color' used uninitialized

Failed to compile fragment shader:
0:12(15): error: no function with name 'texture2D'

terminate called after throwing an instance of 'std::runtime_error'
  what():  The shader shaders/post.vert or shaders/post.frag could not be loaded
[1]    19847 IOT instruction (core dumped)  ./build/2D_fractals_generator
```

Any ideas ?

**Comment by angeluriot** [comment]
Sorry I don't have this issue, feel free to make a pull request if you find the solution.

**Comment by 301ASALA** [comment]
Bonjour, quand j'exécute le .exe le fichier se ferme instantanément, j'ai essayé de le désinstaller et réinstaller plusieurs fois mais rien n'y fait

**Comment by abadiet** [comment]
I got the same issue. Have you found a fix @UncleSamulus ?

**Comment by abadiet** [comment]
Fixed with Dimension3D's [PR#10](https://github.com/angeluriot/Dimension3D/pull/10). Please @angeluriot do the change in the repo Dimension3D.

**Comment by angeluriot** [comment]
Done 👍
